src/run_model.jl

```diff
@@ -39,4 +39,35 @@ function run_model(::Type{T},P::Parameter) where {T<:AbstractFloat}
 
     return Prog
 
+end
+
+function model_setup(::Type{T}=Float32;     # number format
+    kwargs...                             # all additional parameters
+    ) where {T<:AbstractFloat}
+
+    P = ShallowWaters.Parameter(T=T;kwargs...)
```

Comment:
Maybe you want to take out the `ShallowWaters.` declarations as they are only necessary when you defined this outside of the `ShallowWaters` scope